doc/news/changes.h

```diff
@@ -159,6 +159,12 @@ inconvenience this causes.
 <a name="general"></a>
 <h3>General</h3>
 <ol>
+  <li> New: Triangulation::ghost_neighbors() returns a list of MPI ranks of owners
```

Comment:
The data is a list (or array if you will) of unique ranks (so a set in mathematical terms), which I happen to return in a std::vector (not in a std::list of course!) because I think that is the most flexible data structure to use. I thought about using a std::set here and I am willing to change it if you prefer. I generally prefer std::vector over std::set because it is more efficient/faster, compact, I can memcpy data out, etc..
std::set or std::vector?